### DISCUSSION TITLE
Borders

### DISCUSSION THREAD

**Discussion by eduardonwa** [tailwindlabs/tailwindcss#3621] [Help]
Im using the CDN, how can I add borders? Thanks, this doesn't display nothing. Why? Thanks.

```
    <div>
        <div class="border-t-2 border-red-600">
            i live here
        </div>
    </div>
```

**Comment by rotorbit69** [comment]
Try a larger size 

border-t-8 or border-8

**Comment by simonswiss** [comment]
Hey @eduardonwa - that should definitely work.

Are you able to share a mini repo with a reproduction of your problem?

**Comment by eduardonwa** [comment]
I ended up using custom css, thank you for the replies! 👍
